pkg/apis/build/v1alpha1/build_source.go

```diff
@@ -4,15 +4,34 @@
 
 package v1alpha1
 
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
 // BuildSourceType enumerates build source type names.
 type BuildSourceType string
 
+// LocalCopy defines a source type that waits for user input, as in a local copy into a POD.
+const LocalCopy BuildSourceType = "LocalCopy"
```

Comment:
I think we should have HTTP as well now that we introduce the type ?

Reply:
Please consider the latest changes, I've the `type` attribute support for Remote Artifacts as well.